maxim/logger/openai/completions.py

```
@@ -91,7 +99,16 @@ def create(self, *args, **kwargs):
                 f"[MaximSDK][MaximOpenAIChatCompletions] Error in generating content: {str(e)}"
             )
 
-        response = super().create(*args, **kwargs)
+        try:
+            response = super().create(*args, **kwargs)
+        except Exception as e:
+            if generation is not None:
+                generation.error({ "message": str(e) })
+            scribe().warning(
+                f"[MaximSDK][MaximOpenAIChatCompletions] Error in generating content: {str(e)}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)